mozilla-services/consvc-shepherd
# [SHEP-127] Migrate django test cases to use factory for shepherd

## References

[SHEP-127](https://mozilla-hub.atlassian.net/browse/SHEP-127)

## Problem Statement

Currently Django test cases involve manually adding data to the tables and passing that data into the test functions. To streamline the process and reduce repetitive code, we will migrate the test cases to use a factory for generating the required data dynamically.

## Proposed Changes

Added factories for the models and used them to generate the data to use in the test cases.

## Verification Steps

You can run the `make test` command to see if all tests are working as expected after the code changes.


[SHEP-127]: https://mozilla-hub.atlassian.net/browse/SHEP-127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
